### ISSUE TITLE
Make sections in MaterialPopupMenu public

### ISSUE THREAD

**Issue by SasLuca** [zawadz88/MaterialPopupMenu#55]
I want to be able to modify the UI elements in the popup menu when the user clicks on them, however I can't do this right now because they private.

I think it would be better if they were public so that we could modify them or offer users a different way to modify them.

**Comment by zawadz88** [comment]
Hi @SasLuca,
I'm not sure if I understand this. Would like to be able to update the popup menu while it's still displayed without closing it? If so, how would it look like? What kind of modification do you have in mind?

**Comment by SasLuca** [comment]
Yeah, I would like to update the popup menu while it's still displaying without closing it. I now realise that is a bit harder to do. 

Maybe there could be a refresh() function for MaterialPopupMenu that you would call within a callback like such:

```kotlin
var ct = 0

val popupMenu = popupMenu()
{
    section()
    {
        item()
        {
            label = "Counter: $ct"
            callback =
            {
                ct++
                label = "Counter: $ct"
                popupMenu.refresh()
            }
        }
    }
}
```

**Comment by zawadz88** [comment]
If you would like to refresh an individual label when clicked you could use a custom item, get a reference to a textview in that View and update it accordingly when clicked. Or do you have something else in mind?
